api/src/main/kotlin/com/okeicalm/simpleJournalEntry/handler/query/AccountQuery.kt

```diff
@@ -10,4 +10,8 @@ class AccountQuery(private val repository: AccountRepository) : Query {
     fun allAccounts(): List<AccountType> {
         return repository.findAll().map { AccountType(it) }
     }
+
+    fun allAccountsByUserID(user_id: ID): List<AccountType> {
```

Comment:
userIdでfilterしてるので関数名にallつけるのは適切ではないのでは？

Reply:
I named this function name with the intention of referring to all the accounts a user has, but it may be confusing.

I'll try something different and better.